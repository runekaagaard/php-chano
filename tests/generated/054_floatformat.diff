001- 18.13
001+ Fatal error: Call to undefined method DtlIter::floatformat() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/054_floatformat.php on line 4