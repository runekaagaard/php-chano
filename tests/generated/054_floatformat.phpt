--TEST--
A generated testfile for the "floatformat" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 18.125)));
foreach ($items as $i) echo $i->input->floatformat(2);
--EXPECT--
18.13