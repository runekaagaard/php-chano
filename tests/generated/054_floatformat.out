Fatal error: Call to undefined method DtlIter::floatformat() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/054_floatformat.php on line 4